hack/common.sh

```diff
@@ -29,6 +29,7 @@ LATEST_NOOBAA_DB_IMAGE="quay.io/sclorg/postgresql-15-c9s" # Ref-https://github.c
 KUBE_RBAC_PROXY_FULL_IMAGE_NAME="gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1"
 UX_BACKEND_OAUTH_FULL_IMAGE_NAME="quay.io/openshift/origin-oauth-proxy:4.20.0"
 LATEST_MUST_GATHER_IMAGE="quay.io/ocs-dev/ocs-must-gather:latest"
+LATEST_BLACKBOX_EXPORTER_IMAGE="quay.io/prometheus/blackbox-exporter:latest"
```

Comment:
Do we have a tagged image for this if possible or is it intentional to always use the latest?

Reply:
If this is just for the CI, the latest tag is probably fine. But the downstream build has a list of supported downstream images that we ship with, please confirm with the build team that there is one officially supported by RH.